### DISCUSSION TITLE
Further increase accessibility for a broader audience

### DISCUSSION THREAD

**Discussion by SichangHe** [SichangHe/internet_route_verification#191] [General]
> - Paper assumes the reader is familiar with RPSL

This is tricky. Do we just explain everything more, @cunha?

**Comment by SichangHe** [comment]
> 3) Refine background and statements to be more detailed and accessible, particularily the ones mentioned in review B & E.

**Reply by cunha** [reply]
It's a space issue. Let's ask the shepherd for an extra page in the main body. If he agrees we can add more background and move the appendix into the main body.

**Reply by cunha** [reply]
With no extra page we can just expand the appendix to make it more didatic.
